### PR TITLE
Development ipfs cluster

### PR DESCRIPTION
**Pull request by maayarosama**
### Related Issues
https://github.com/threefoldtech/tf-pinning-service/issues/3

### CODE REVIEW

File: tfgrid3/ipfs_cluster/zinit/sshd.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+exec: bash -c "/usr/sbin/sshd -D"
```
Comment (AhmedHanafy725) [inline tfgrid3/ipfs_cluster/zinit/sshd.yaml:1]: can you please make this service run after sshkey service?

File: tfgrid3/ipfs_cluster/zinit/secret.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+exec: |
+  bash -c '
+  sleep 60
```
Comment (AhmedHanafy725) [inline tfgrid3/ipfs_cluster/zinit/secret.yaml:3]: why `sleep` here if it runs after another service?

File: tfgrid3/ipfs_cluster/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+RUN cp /ipfs-cluster-follow/ipfs-cluster-follow /bin
+
+
+RUN cd go-ipfs && bash install.sh
+
+RUN mkdir -p /etc/zinit
+COPY zinit /etc/zinit
+# COPY ./ipfs.sh /
+
+# RUN  chmod +x ./ipfs.sh
```
Comment (AhmedHanafy725) [inline tfgrid3/ipfs_cluster/Dockerfile]: please remove these commented lines

**Review by AhmedHanafy725 — CHANGES_REQUESTED** [review]
please add in the readme:
- how to build the image 
- how to run it with docker

File: tfgrid3/ipfs_cluster/zinit/secret.yaml (hunk 2 above)
Comment (maayarosama) [reply tfgrid3/ipfs_cluster/zinit/secret.yaml:3]: Because the other service needs time to create the required files used by this service

**Review by AhmedHanafy725 — APPROVED** [review]

**Pull request merged**